File: sdks/thunkmetrc-go/wrapper/models/plants_update_plant_location_request_item.go
package models

type PlantsUpdatePlantLocationRequestItem struct {
    ActualDate string `json:"ActualDate,omitempty"`
    Id int `json:"Id,omitempty"`
    Label string `json:"Label,omitempty"`
    Location string `json:"Location,omitempty"`
    Sublocation string `json:"Sublocation,omitempty"`
}

File: sdks/thunkmetrc-rs/wrapper/src/models/plants_update_plant_location_request_item.rs
use serde::{Deserialize, Serialize};
#[allow(unused_imports)]
use super::*;

#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct PlantsUpdatePlantLocationRequestItem {
    #[serde(rename = "actual_date", skip_serializing_if = "Option::is_none")]
    pub actual_date: Option<String>,
    #[serde(rename = "id", skip_serializing_if = "Option::is_none")]
    pub id: Option<i64>,
    #[serde(rename = "label", skip_serializing_if = "Option::is_none")]
    pub label: Option<String>,
    #[serde(rename = "location", skip_serializing_if = "Option::is_none")]
    pub location: Option<String>,
    #[serde(rename = "sublocation", skip_serializing_if = "Option::is_none")]
    pub sublocation: Option<String>,
}
